1xx-enable-ssl-for-kafka-bridge.md

```diff
@@ -0,0 +1,148 @@
+  http:
+    port: 8443
+    tls: #NEW FIELD
+      serverCertChainAndKey: #NEW FIELD
+        secretName: my-secret
+        certificate: public.crt
+        key: private.key
+      config: #NEW FIELD
+        ssl.enabled.protocols: "TLSv1.1, "TLSv1.2", "TLSv1.3"
+        ssl.enabled.cipher.suites: "TLS_DHE_RSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384"
+...
+```
+
+## Future Improvements
+
+### Support more formats for the keystore
+We can support formats such as `PKSC12` and `JKS` as they are common for Java applications. If we support these formats, then we would need to add the following configurations:
+- http.ssl.keystore.type
+- http.ssl.keystore.password
+
+### Support mTLS
+In the future, we can support mutual TLS for the client connections to HTTP Bridge as well. This could be implemented by adding a field `trustedCertificates`, for example:
+
+```yaml
+apiVersion: kafka.strimzi.io/v1beta2
+kind: KafkaBridge
+metadata:
+  name: my-bridge
+spec:
+  replicas: 3
+  bootstrapServers: <cluster_name>-cluster-kafka-bootstrap:9092
+  http:
+    port: 8443
+    tls: 
+      serverCertChainAndKey: 
+        secretName: my-secret
+        certificate: public.crt
+        key: private.key
+      trustedCertificates: #NEW FIELD
+        - secretName: my-cluster-cluster-cert
+          pattern: "*.crt"
+        - secretName: my-cluster-cluster-cert
+          pattern: "*.crt"    
+...
+```
+This is not in the scope of this proposal, as we will likely to support multiple authentication mechanisms in the future.
+
+In the future, we are also likely to implement authentication and authorization for HTTP Bridge. By having separate listeners, the internal endpoints will not be affected by such changes and will make it easier to implement them.
+
+### Automating certificate management
+
+Users can choose to allow the operator to manage the certificate instead of providing their own certificate and key. This can be implemented by adding another boolean field on the KafkaBridge CR such as `generateCertificate`. The operator then could issue the certificate using the same CA for their Kafka cluster as we do today for internal components such as Cruise Control and Kafka Exporter, however with the upcoming support for cert-manager, the way the operator manages certificates will change. This is the reason we are leaving this feature for future improvement.
+
+## Affected/not affected projects
+
+- strimzi-kafka-bridge
+- strimzi-cluster-operator
+
+## Compatibility
+
+This will introduce a breaking change for users who collect metrics on port 8080 or the port defined by `http.port` configuration. Also users who run Bridge on baremetals and do their own manual healtchecks using `/healthy` and `/ready` endpoints will be affected as well. These users have to reconfigure their healthcheck and monitoring system to use port 8081 or reconfigure their Bridge server with `http.management.port`.
```

Comment:
```suggestion
This will introduce a breaking change for users who collect metrics on port 8080 or the port defined by `http.port` configuration. Also users who run the HTTP Bridge on bare metal and do their own manual health checks using `/healthy` and `/ready` endpoints will be affected as well. These users have to reconfigure their health check and monitoring system to use port 8081 or reconfigure their Bridge server with `http.management.port`.
```